OpenKeychain/src/main/java/org/sufficientlysecure/keychain/securitytoken/SecurityTokenInfo.java

```diff
@@ -74,9 +80,41 @@ public static SecurityTokenInfo newInstanceDebugLockedHard() {
         if (!BuildConfig.DEBUG) {
             throw new UnsupportedOperationException("This operation is only available in debug builds!");
         }
-        return SecurityTokenInfo.create(
+        return SecurityTokenInfo.create(TransportType.NFC, TokenType.UNKNOWN_NFC,
                 new byte[][] { KeyFormattingUtils.convertFingerprintHexFingerprint("4700BA1AC417ABEF3CC7765AD686905837779C3E") },
                 Hex.decode("010203040506"), "yubinu2@mugenguild.com", "http://valodim.stratum0.net/mryubinu2.asc", 0, 0);
     }
 
+    public enum TransportType {
+        NFC, USB
+    }
+
+    public enum TokenType {
```

Comment:
Is there a particular reason you put UNKNOWN_NFC and UNKNOWN_USB here redundantly? This information is already available from the TransportType.